src/shims/x86/mod.rs

```diff
@@ -468,6 +468,79 @@ fn unary_op_ps<'tcx>(
     Ok(())
 }
 
+enum ShiftOp {
```

Comment:
Please add doc comments explaining these. "Sll" is not exactly a self-explaining acronym.